src/model_signing/_signing/sign_certificate.py

```diff
@@ -222,8 +222,12 @@ def _to_openssl_certificate(certificate_bytes, log_fingerprints):
 
         extensions = signing_certificate.extensions
         can_use_for_signing = False
-        usage = extensions.get_extension_for_class(x509.KeyUsage)
-        if usage.value.digital_signature:
+        try:
+            usage = extensions.get_extension_for_class(x509.KeyUsage)
+        except x509.ExtensionNotFound:
+            usage = None
```

Comment:
Alternatively we could throw an error here not accepting certs without KeyUsage.

Reply:
I think at least logging a warning here would be good. Similarly, for the `pass` on line 240, logging would also be good there.

Then, rather than having `usage` be both a valid instance and `None`, maybe we can move the `if usage.value.digital_signature` inside the try block and change the `else` on 232 to be `if not can_use_for_signing`.

And now I realize that looking for `x509.KeyUsage` and `x509.ExtendedKeyUsage` is almost similar.